### PR TITLE
[4.4] Postinstallation-Message htaccess.txt brotli change

### PR DESCRIPTION
**Pull request by MacJoom**
Postinstallation-Message for Brotli compression bug #43055 

Pull Request for Issue # .

### CODE REVIEW

File: administrator/components/com_admin/postinstall/htaccessbrotli.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_admin
+ *
+ * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
```
Comment (Quy) [inline administrator/components/com_admin/postinstall/htaccessbrotli.php]:
```suggestion
 * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
```

**Comment by brianteeman** [comment]
not sure what will happen when someone who has updated to 4.4.4 then goes to 5.1 - worse case i _think_ is that they will see the message twice

**Comment by richard67** [comment]
> not sure what will happen when someone who has updated to 4.4.4 then goes to 5.1 - worse case i _think_ is that they will see the message twice

@brianteeman Check my comments in the other PR. The message will not be duplicated as we have the `INSERT IGNORE`/ÒN CONFLICT DO NOTHING`.

**Comment by brianteeman** [comment]
which is why I said "not sure" ;)

**Pull request merged**